### ISSUE TITLE
RIDE keeps asking for confirmation of file changes

### ISSUE THREAD

**Issue by nicolas-dyalog** [Dyalog/ride#610]
### Describe the issue you are having
Unlike the IDE, it does not allow the user to choose not to be asked again.
https://user-images.githubusercontent.com/10961427/57489882-54428680-72af-11e9-9f35-b66ea724c7ba.png

### How do you reproduce the issue?
(⊂':Class MyClass' ':EndClass')⎕NPUT '/tmp/myclass.aplc' 1
⎕ED 'MyClass' 
⍝ make some changes in editor and fix. The dialog box will pop up.

Originally reported by Adam in https://github.com/Dyalog/link/issues/54

**Comment by nicolas-dyalog** [comment]
mkromberg commented on May 27, 2019

I think this is because the switch is stored in the registry and RIDE does not have access to write to it - we need to either extent the RIDE API so RIDE can ask the interpreter to do this, or to allow RIDE to manage its own settings. This should be reviewed in the context of Dyalog APL v18.0, which will probably have a new way to manage configuration.

As a work-around, try saving the setting using the IDE.

**Comment by nicolas-dyalog** [comment]
e9gille commented on May 27, 2019

RIDE is already managing its own settings, so extending that wouldn't be a problem.

I'm sure I have discussed this before with Jay when I raised it as an issue that the interpreter doesn't list an option to remember the response. Jay felt that was up to the RIDE client to keep track of, but I never did anything about it.

If the client was to offer to remember the answer, like in the ODE, it would presumably need to identify the question and options offered (using the title perhaps?). The answer could be stored with the other settings and cleared from the "Preferences" UI (just like in the ODE).
